noir-projects/aztec-nr/aztec/src/macros/functions/stub_registry.nr

```diff
@@ -0,0 +1,19 @@
+use std::{collections::umap::UHashMap, hash::{BuildHasherDefault, poseidon2::Poseidon2Hasher}};
```

Comment:
This got moved in here from `interfaces.nr`. The other change here is that instead of exposing the STUBS  global I just expose the setter and getter. This makes it nicer to read as everywhere we either use only the getter or the setter.

Reply:
lovely